shadow_removal.py

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
What is this script doing?

Reply:
It was just some of our testing code for using YUV. Should I delete it?